dali/base/dasds.cpp

```diff
@@ -2809,6 +2809,8 @@ class CNodeSubscriptionManager : public CSimpleInterface, implements INodeSubscr
     // ISubscriptionManager impl.
     virtual void add(ISubscription *sub, SubscriptionId id)
     {
+        CHECKEDDALIREADLOCKBLOCK(owner.dataRWLock, readWriteTimeout);
+        CHECKEDCRITICALBLOCK(owner.treeRegCrit, fakeCritTimeout);
         CriticalBlock b(lock);
```

Comment:
Yes to protect subscribersById/subscriberListByNode.

can rename, but it's the only crit in CNodeSubscriptionManager, I wouldn't normally name general crit in class like this..
I'll rename it "subscriberListCrit"